resources/views/components/menu/main.blade.php

```diff
@@ -77,7 +77,7 @@
             <x-dropdown-header>Miscellaneous</x-dropdown-header>
             <x-dropdown-item :href="route('game.index')">All Games</x-dropdown-item>
             {{--<x-dropdown-item href="popularGames.php">Most Played</x-dropdown-item>--}}
-            <x-dropdown-item :href="url('gameSearch.php?p=0')">Hardest Games</x-dropdown-item>
```

Comment:
Not sure how it's working without an explicit redirect, but hitting the old URL (http://localhost:64000/gameSearch.php?p=0) redirects to the All Games page without a sort.

Reply:
Fixed in latest.